modules/hitl/src/views/full/components/messages/Message.tsx

```diff
@@ -122,12 +29,19 @@ export default class Message extends React.Component<{ message: HitlMessage }> {
 
     let messageFrom = 'bot'
     if (direction === 'in') {
+      // TODO: Visit and session reset are currently unsupported by @botpress/messaging-components
       if (type === 'visit') {
         return (
```

Comment:
Should we even display those?

Reply:
I think it could be relevant for the agent, knowing when was the last time a user visited the bot.